ansible/roles/openshift_master_resource_quota/README.md

```diff
@@ -0,0 +1,45 @@
+openshift_master_storage_class
+=========
+
+A role that deploys quotas for Openshift cluster resources
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+osmsc_enable_quotas: BOOL.  Should quotas be enabled.
```

Comment:
we try to name the role variables based on an abbreviation of the role name.

for openshift_master_storage_class = osmsc.

For this new role, probably want to name it: osmrq?